src/java.base/share/classes/jdk/internal/util/DecimalDigits.java

```diff
@@ -132,6 +134,54 @@ public static int stringSize(long x) {
         return 19 + d;
     }
 
+    /**
+     * Determine whether the two character in str are both digits. If they are, return (str[off] - '0') * 10 + (str[off] - '0'), otherwise return -1
```

Comment:
`off` -> `offset`
The second index should be `offset + 1`.
Same in the `@return` clause.